inbox/logging.py

```diff
@@ -209,7 +210,12 @@ class BoundLogger(structlog.stdlib.BoundLogger):
     """BoundLogger which always adds greenlet_id and env to positional args"""
 
     def _proxy_to_logger(self, method_name, event, *event_args, **event_kw):
-        event_kw["greenlet_id"] = id(gevent.getcurrent())
+        from inbox.config import config
+
+        if config.get("USE_GEVENT", True):
+            event_kw["greenlet_id"] = id(gevent.getcurrent())
+        else:
+            event_kw["thread_id"] = threading.get_ident()
```

Comment:
Note that this piece is shared between API and sync pods, sync pods still use gevent at the time of writing.